apis/v1/backendtlspolicy_types.go

```diff
@@ -88,6 +88,14 @@ type BackendTLSPolicySpec struct {
 	// implementation MUST ensure the `Accepted` Condition is set to
 	// `status: False`, with Reason `Conflicted`.
 	//
+	// Because of concerns over recording status correctly in some edge cases,
+	// currently, implementations SHOULD support only a single reference
+	// in the `targetRefs` list. This support will be expanded in a future release,
+	// once we have updated the conflict resolution and status handling
+	// to cover the edge cases.
+	//
+	// Implementations SHOULD NOT support more than one targetRef (again, for now).
```

Comment:
The first and second paragraph feel a bit repetitive here, maybe it could be simplified to something like this?

```suggestion
	// Implementations SHOULD NOT support more than one targetRef at this
	// time. Although the API technically allows for this, the current guidance
	// for conflict resolution and status handling is lacking. Until that can be 
	// clarified in a future release, the safest approach is to support a single
        // targetRef. 
```